CorsixTH/Lua/dialogs/resizables/hotkey_assign.lua

```diff
@@ -764,328 +630,67 @@ function UIHotkeyAssign:close()
   end
 end
 
-function UIHotkeyAssign:toggleButton()
-  self:close()
-  local window = UIHotkeyAssign_Panels(self.ui, "menu")
-  self.ui:addWindow(window)
-end
-
-function UIHotkeyAssign:storeRecallPosButton()
-  self:close()
-  local window = UIHotkeyAssign_storeRecallPos(self.ui, "menu")
-  self.ui:addWindow(window)
+function UIHotkeyAssign:showKeyPane(pane)
+  self.key_pane = pane
+  for k, v in pairs(self.key_windows) do
+    v.window:setVisible(k == pane)
+    v.button:setToggleState(k == pane)
+  end
 end
 
 function UIHotkeyAssign:confirm_func(hotkey)
   hotkey_input(hotkey, self.hotkey_buttons, self.app)
 end
 
---! Assign panel keys window.
-class "UIHotkeyAssign_Panels" (UIResizable)
-
----@type UIHotkeyAssign_Panels
-local UIHotkeyAssign_Panels = _G["UIHotkeyAssign_Panels"]
+class "UIHotkeyAssignKeyPane" (Window)
+local UIHotkeyAssignKeyPane = _G["UIHotkeyAssignKeyPane"]
```

Comment:
This gets through lua doc and/or luacheck?

Afaik, the 4 lines to define a class are checked somewhere for their form

Reply:
Fixed.

As far as I can tell nothing actually requires or even uses @type - but it's consistent.